doc/examples/viz_horizon.py

```diff
@@ -0,0 +1,278 @@
+====================================================
+
+This tutorial demonstrates how to use DIPY Horizon programmatically in Python
+code for interactive visualization of tractography data, brain images,
+surfaces, and peak directions. Horizon is a powerful interactive medical
+visualization tool introduced in :footcite:t:`Garyfallidis2019`.
+
+While Horizon can be used from the command line (see :ref:`viz_flow`), this
+example shows how to use it directly in Python scripts for more flexible
+and programmatic visualization workflows.
+
+What is Horizon?
+----------------
+
+Horizon is DIPY's interactive visualization system that allows you to:
+
+- Visualize and interact with tractograms (streamlines)
+- Display anatomical images (T1, T2, FA, etc.) as slices
+- Cluster streamlines using QuickBundlesX
+- Visualize peak directions and spherical harmonics
+- Display brain surfaces (cortical meshes)
+- Combine multiple data types in a single view
+- Save snapshots of your visualizations
+
+Getting Started
+---------------
+
+First, let's import the necessary modules and check that we have FURY
+installed (required for 3D visualization):
+
+"""
+
+import sys
+
+import numpy as np
+
+from dipy.core.gradients import gradient_table
+from dipy.data import default_sphere, get_fnames
+from dipy.direction import peaks_from_model
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.io.image import load_nifti
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.reconst.csdeconv import (
+    ConstrainedSphericalDeconvModel,
+    auto_response_ssst,
+)
+from dipy.reconst.dti import TensorModel
+from dipy.segment.mask import median_otsu
+from dipy.tracking import utils
+from dipy.tracking.stopping_criterion import ThresholdStoppingCriterion
+from dipy.tracking.streamline import Streamlines
+from dipy.tracking.tracker import deterministic_tracking
+from dipy.utils.logging import logger
+from dipy.utils.optpkg import optional_package
+from dipy.viz import has_fury
+
+fury, has_fury, setup_module = optional_package("fury")
+
+interactive = False  # Set to True for interactive visualization!
+
+###############################################################################
+
+# Check if FURY is available
+if not has_fury:
+    logger.error("FURY is required for Horizon. Install it with: pip install fury")
+    sys.exit(1)
+
+from dipy.viz import horizon
+
+###############################################################################
+# Basic Example: Visualizing a Brain Image
+# =========================================
+#
+# Let's start with the simplest use case: visualizing a 3D brain image.
+# Load example diffusion data (Stanford HARDI)
+# We'll extract and visualize its b0 volume (first 3D volume of the 4D data).
+
+# Load example data
+hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames(name="stanford_hardi")
+data, affine, hardi_img = load_nifti(hardi_fname, return_img=True)
+
+# Extract b0 image (first volume)
```

Comment:
this data have multiple B0.  Update the comment saying you use the first b0 image